tests/unit/cluster/multi-slot-operations.tcl

```diff
@@ -107,3 +107,15 @@ test "DELSLOTSRANGE command with several boundary conditions test suite" {
     assert_match "*9829 11000*12001 12100*12201 13104*" [$master4 CLUSTER SLOTS]
 }
 } cluster_allocate_with_continuous_slots_local
+
+start_cluster 1 0 {tags {external:skip cluster}} {
+    test "Regression test for multi-exec with RANDOMKEY accessing the wrong per-slot dictionary" {
+        R 0 SETEX FOO 10000 BAR
+        R 0 SETEX FIZZ 10000 BUZZ
+
+        R 0 MULTI
+        R 0 GET FOO
+        R 0 RANDOMKEY
+        R 0 EXEC
```

Comment:
This test suite is about admin commands like CLUSTER ADDSLOTRANGE, etc. I don't think it was intended for things like RANDOMKEY in MULTI.

RANDOMKEY is tested in unit/keyspace.tcl. MULTI is tested in unit/multi.tcl. One of those would be a better place for this test case IMO.

Reply:
The issue is specific to cluster mode enabled. Do we have the capability to enable it in unit/keyspace.tcl?

Reply:
Yea i believe it's possible. Just add a tag I think. I'll check tomorrow.

Reply:
I can play around with the test. Maybe something like this would be deterministic
```
        R 0 SETEX FOO 10000 BAR
        R 0 SETEX FIZZ 10000 BUZZ
        R 0 MULTI
        R 0 DEL FOO
        R 0 RANDOMKEY
        R 0 EXEC /* + verification */
```

The cached slot will be of FOO and then RANDOMKEY must select FIZZ.

Reply:
I see test cases under `unit/` use this style to run in cluster mode:

```tcl
start_cluster 1 0 {tags {"external:skip cluster"}} {
    test "Bla bla bla" {
        r flushall
        # ...        
    }
}
```

Reply:
Sweet, moved it to multi.tcl and used your configuration and it seems like it produces a consistent crash